packages/nx/src/plugins/js/index.ts

```diff
@@ -22,7 +22,7 @@ export const processProjectGraph: ProjectGraphProcessor = async (
   graph,
   context
 ) => {
-  const builder = new ProjectGraphBuilder(graph);
```

Comment:
Is this the manifestation of a breaking change for existing plugins? e.g. In `nx-dotnet`, I create a project graph builder. If the fileMap arg here is required, after this is published my plugin breaks.